src/IECoreHoudini/SceneCacheNode.cpp

```diff
@@ -244,17 +235,11 @@ OP_TemplatePair *SceneCacheNode<BaseType>::buildOptionParameters()
 		);
 		
 		thisTemplate[5] = PRM_Template(
-			PRM_TOGGLE | PRM_TYPE_JOIN_NEXT, 1, &pTagGroups, &tagGroupsDefault, 0, 0, 0, 0, 0,
+			PRM_TOGGLE, 1, &pTagGroups, 0, 0, 0, 0, 0, 0,
 			"Convert SCC tags into Houdini primitive groups."
 		);
 
 		thisTemplate[6] = PRM_Template(
-			PRM_STRING, 1, &pTagGroupsPrefix, 0, 0, 0, 0, 0, 0,
-			"Names for groups created from tags will start with this prefix.",
-			&cTagGroupsOptions
-		);
-
-		thisTemplate[7] = PRM_Template(
```

Comment:
For some reason changing this to 7 causes a segfault when the node is created in Houdini.